finance-api/user-guides/fetch-account-transactions.md

```diff
@@ -0,0 +1,184 @@
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc). 
+
+## Fetch transactions during a specific period
+When fetching transactions from a merchant's Zettle account during a specific period, set pagination to avoid a big dataset in a response. The transactions should be fetched from the merchant's liquid account.
+
+
+1. Send a request where you set `limit` as the number of transactions to fetch and `offset` as `0`.
+     
+   ```
+   GET /organizations/self/accounts/{accountTypeGroup}/transactions?start={start_time}&end={end_time}&{limit}={limit_value}&offset={offset_value}
+   ```
+
+   Example:
+   
+   The following example fetches the latest three transactions from the merchant's liquid account from 1 January, 2020 to 5 July, 2020. 
+   
+   Request   
+   ```
+   GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-07-05&limit=3&offset=0
+   ```
+   Response         
+   ```json
+       {
+           "data": [
+               {
+                   "timestamp": "2020-07-04T20:16:44.309+0000",
+                   "amount": 381,
+                   "originatorTransactionType": "CARD_PAYMENT_FEE_REFUND",
+                   "originatingTransactionUuid": "30cef6e2-be09-11ea-a8e4-bce028663c34"
+               },
+               {
+                   "timestamp": "2020-07-04T20:16:44.309+0000",
+                   "amount": -20610,
+                   "originatorTransactionType": "CARD_REFUND",
+                   "originatingTransactionUuid": "30cef6e2-be09-11ea-a8e4-bce028663c34"
+               },
+               {
+                   "timestamp": "2020-06-27T23:52:18.327+0000",
+                   "amount": 649,
+                   "originatorTransactionType": "PAYMENT_FEE", // In case of refund
+                   "originatingTransactionUuid": "690c99ea-b6ef-11ea-9730-7ef7aeff642d"
+               },
+           ]
+       }
+   ```
+   
+2. Send another request where you keep `limit` the same as in step 1 and increment `offset` with the value of `limit` . 
+     
+   ```
+   GET /organizations/self/accounts/{accountTypeGroup}/transactions?start={start_time}&end={end_time}&{limit}={limit_value}&offset={offset_value}
+   ```
+   Example:
+      
+   The following example fetches the next three transactions from the merchant's liquid account from 1 January, 2020 to 5 July, 2020.
+   
+   Request
+    
+   ```
+   GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-07-05&limit=3&offset=3
+   ```
+   Response
+   
+    ```json
+    {
+        "data": [
+            {
+                "timestamp": "2020-06-27T23:52:18.327+0000",
+                "amount": -35100,
+                "originatorTransactionType": "PAYMENT", // In case of refund
```

Comment:
I didn't get this example. Are you trying to say that if the payment `45f51ed4-b7bf-11ea-90de-435a3f6e4738` was refunded we would show this?

Reply:
This would probably be better like this to show that amount is negative in case of refund
 {
                "timestamp": "2020-06-27T23:52:18.327+0000",
                "amount": -35100, // Negative in case of refund
                "originatorTransactionType": "PAYMENT",